wodles/aws/aws_s3.py

```diff
@@ -2624,7 +2623,8 @@ def marker_only_logs_after(self, aws_region: str, aws_account_id: str) -> str:
     def check_bucket(self):
         """Check if the bucket is empty or the credentials are wrong."""
         try:
-            if not 'CommonPrefixes' in self.client.list_objects_v2(Bucket=self.bucket, Delimiter='/'):
+            bucket_objects = self.client.list_objects_v2(Bucket=self.bucket, Delimiter='/')
+            if not 'CommonPrefixes' in bucket_objects and not 'Contents' in bucket_objects:
```

Comment:
Check if `Content` isn't an empty list because, according to the [documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html?highlight=list_objects_v2#S3.Client.list_objects_v2), this key always is present.
```suggestion
            if not 'CommonPrefixes' in bucket_objects and not bucket_objects['Contents']:
```